### PR TITLE
Would like an easy way to extend the IBM FHIR Server TransactionManager timeout interval #1660

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: docs/src/pages/guides/FHIRServerUsersGuide.md [hunk 1]
```diff
@@ -432,6 +432,11 @@ For a Derby-related datasource definition, any bean property supported by the `E
 
 To disable the multitenant feature for a particular offering add to your `fhirServer/persistence/datasources` entry `multitenant` and set false to disable, and true to enable, only for Db2 is the default set to true.
 
+#### 3.4.2.4 Database Access TransactionManager
+The TransactionManager controls the timeout of database queries.  
+
+To expand the transaction timeout value, one can copy over the `transaction-manager-long.xml` from the WLP configDropins from `/disabled` to `/overrides` folder, or set the Environment variable `FHIR_TRANSACTION_MANAGER=120s` or enter the value in the server.env file at the root of the WLP instance.  The value should be at least as granular as seconds or minutes.  Example values are 120s or 2m.  You should not lower this below 120s.
```
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:438]: Are you sure the liberty server.xml "variables" are automatically valued from environment variables?
I thought you needed to use `${env.VAR_NAME}` for that.

**Review by lmsurpre — COMMENTED** [review]
I'm wondering if the variable should have the word "timeout" in it. `FHIR_TRANSACTION_TIMEOUT` or something like that?

File: docs/src/pages/guides/FHIRServerUsersGuide.md (hunk 1 above)
Comment (prb112) [reply docs/src/pages/guides/FHIRServerUsersGuide.md:438]: I tried it out locally. It seemed to work for me.

**Comment by prb112** [comment]
> I'm wondering if the variable should have the word "timeout" in it. `FHIR_TRANSACTION_TIMEOUT` or something like that?

Sure - if this is what Dag is looking for, I can make the change.

**Review by lmsurpre — APPROVED** [review]
LGTM

**Pull request merged**